ts/features/messages/utils/strings.ts

```diff
@@ -19,3 +22,11 @@ export const getBadgeTextByPaymentNoticeStatus = (
       return "";
   }
 };
+
+export const getHeaderByKey = (headers: Record<string, string>, key: string) =>
+  pipe(
+    Object.entries(headers),
+    A.findFirst(([k, _]) => k.toLowerCase() === key),
```

Comment:
```suggestion
    A. findFirstMap(([key, value]) => key.toLowerCase() === key ? O.some(value) : O.none),
```

Reply:
It should be indifferent, do you agree?

Reply:
It is the same but when using `A.findFirstMap` we can avoid having to `O.map` later. Basically, we can do two operations (`A.findFirst` and `O.map`) in one (`A.findFirstMap`). What do you think?

Reply:
Changed https://github.com/pagopa/io-app/pull/5149/commits/170c51c8c6e2912bfa75135edfd7c67dcd6a9128